browser/js/admin/admin.controller.js

```diff
@@ -0,0 +1,34 @@
+app.controller('AdminCtrl', function($scope, $state, $mdSidenav, $mdMedia, ProductFactory, getAllProducts) {
+
+  $scope.imagePath = 'assets/images/placeholder.jpg';
+
+  $scope.products = getAllProducts
+
+  $scope.deleteProduct = function(product, index){
```

Comment:
This logic should happen inside the factory, consider using a cache in the factory to store the products